examples/solverdummies/cpp/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ project(SolverdummyCPP LANGUAGES CXX DESCRIPTION "preCICE CPP solver dummy")
 find_package(precice REQUIRED CONFIG)
 
 add_executable(solverdummy solverdummy.cpp)
-set_target_properties(solverdummy PROPERTIES CXX_STANDARD 11)
+set_target_properties(solverdummy PROPERTIES CXX_STANDARD 17)
```

Comment:
Is it intended that this update is part of this PR?

Reply:
Yes, we need C++17 for `std::string_view`